src/main/java/org/terasology/launcher/settings/BaseLauncherSettings.java

```diff
@@ -39,7 +35,7 @@
  * @deprecated to be replaced by {@link org.terasology.launcher.config.Config}
  */
 @Deprecated
-public final class BaseLauncherSettings extends AbstractLauncherSettings {
+public final class BaseLauncherSettings extends LauncherSettings {
```

Comment:
Next up for extraction are the `init` methods. As far as I have seen these contain some logic to actually _apply_ the settings, like changing a locale. However, I'm not sure about the details, will have another look...